soccer/gameplay/robocup-py.cpp

```diff
@@ -94,6 +95,11 @@ void OurRobot_move_to_direct(OurRobot* self, Geometry2d::Point* to) {
     if (to == nullptr) throw NullArgumentException("to");
     self->moveDirect(*to);
 }
+void OurRobot_move_to_direct_end_vel(OurRobot* self, Geometry2d::Point* to,
```

Comment:
If I use the normal move function it tries to path plan around the ball instead of moving with it.